error in `yarn test` in endo master

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ses@0.15.1 test: `yarn build && ava`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ses@0.15.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/markmiller/.npm/_logs/2021-12-08T04_44_12_584Z-debug.log

lerna ERR! npm run test exited 1 in 'ses'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I'm not able to reproduce the problem.

I did discover that we lack `CONTRIBUTING.md` clues.

Details:

On this Ubuntu 18.04 box, I made a fresh clone of b5cd6a560b4e0a21f2de4f59027b58473f94dd1e and used an LTS version of node:

```
connolly@jambox:/tmp/endo$ nvm use --lts
Now using node v16.13.0 (npm v8.1.0)
```

I ran `yarn` and then `yarn test` and got:
```
  Uncaught exception in test/test-static-module-record.js

  Error: Cannot find module '/tmp/endo/node_modules/ses/dist/ses.umd.js' imported from /tmp/endo/packages/static-module-record/test/lockdown.js
```

I tried to look for clues on how to run the tests in `CONTRIBUTING.md` but there is no such file.
I don't see anything about how to run tests in `README.md` either.

so let's try `yarn build && yarn test` based on shop lore (I thought it would be documented in [agoric sdk unit testing](https://github.com/Agoric/agoric-sdk/wiki/agoric-sdk-unit-testing) but it's not there either). It seems to work fine:

```
lerna success run Ran npm script 'test' in 13 packages in 16.7s:
lerna success - @endo/base64
lerna success - @endo/cjs-module-analyzer
lerna success - @endo/compartment-mapper
lerna success - @endo/endo
lerna success - @endo/eslint-config
lerna success - @endo/eslint-plugin
lerna success - @endo/netstring
lerna success - @endo/ses-ava
lerna success - @endo/ses-types-test
lerna success - ses
lerna success - @endo/static-module-record
lerna success - @endo/syrup
lerna success - @endo/zip
Done in 17.00s.
```

--- a/packages/ses/test/test-console-error-trap.js
+++ b/packages/ses/test/test-console-error-trap.js
@@ -13,7 +13,7 @@ const exitAssertions = (t, expectedCode, altExpectedCode = expectedCode) => {
     // depending on the platform. Truncation normalizes both of these to the
     // same expected code.
     // eslint-disable-next-line no-bitwise
-    const code = err.code & 0xff;
+    const code = err.code === null ? null : err.code & 0xff;
     t.assert(
       code === expectedCode || code === altExpectedCode,
       'exit error code',

